legal-api/src/legal_api/models/alternate_name.py

```diff
@@ -281,6 +281,11 @@ def alias_json(self):
         alias = {"id": str(self.id), "name": self.name, "type": self.name_type.name}
         return alias
 
+    @property
+    def founding_date(self):
+        """Return the founding_date."""
+        return self.start_date
+
```

Comment:
Don't think we want to do this.  `start_date` can be used for name translations and other future name types which isn't a founding date.

Reply:
Then, which value should I use as founding_date for Alt_names
I referred to this code, line 297

https://github.com/kzdev420/lear/blob/1d171cb28fc5044cbe40014ea89a3e8eb20a4266/legal-api/src/legal_api/models/alternate_name.py#L284-L306

Reply:
You can just use `start_date` directly.  

If you want to really add the `founding_date` property, then add it to `BusinessCommon`.  But you will need to update the LE model's `founding_date` property to be `_founding_date` so we an set the value.

Reply:
Got it

Reply:
I used the `start_date` directly

Reply:
where are you using `start_date` directly?  can you link me?

Reply:
https://github.com/bcgov/lear/blob/1b40e6d07ba8b1d090b4ffd21ca3aaef102f1533/legal-api/src/legal_api/reports/report.py#L466-L468